src/components/ControlBar/ControlBar.tsx

```diff
@@ -17,11 +18,17 @@ function ControlBar(props: ControlBarProps) {
     ctaButtonText,
     nextButtonHandler,
     isNextButtonDisabled,
+    instructions,
   } = props;
 
   return (
     <div className='cio-question-buttons-container'>
       {showBackButton && <BackButton onClick={backButtonHandler} />}
+      {instructions ? (
```

Comment:
We should be able to clip this even further like Islam mentioned 
```
{instructions && <div className='cio-question-buttons-container-text'>{instructions}</div>}
```
Not a big deal, it just has a bit better readability

Reply:
I need to burn this pattern into my brain. Thanks for the callout!

Reply:
Not a must have, just makes it more concise